starter_code/elevator.js

```diff
@@ -1,19 +1,70 @@
 class Elevator {
-  constructor(){
-    this.floor      = 0;
-    this.MAXFLOOR   = 10;
-    this.requests   = [];
-  }
-
-  start() { }
-  stop() { }
-  update() { }
-  _passengersEnter() { }
-  _passengersLeave() { }
-  floorUp() { }
-  floorDown() { }
-  call() { }
-  log() { }
+    constructor() {
+        this.floor = 0;
+        this.MAXFLOOR = 10;
+        this.MINFLOOR = -3;
+        this.requests = [];
+        this.waitingList = [];
+        this.passengers = [];
+        this.direction = null;
+    }
+
+     _start() {
+        return setInterval(() => {
+            this.update();
+        }, 1000);
+    }
+
+    stop() {
+        clearInterval(this._start());
```

Comment:
Here you would call this._start again, and then you would clean the interval for that start, not the previous one.